lab_5_scrapper/scrapper.py

```diff
@@ -168,6 +298,13 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        headline = article_soup.find(itemprop="headline")
+        text_elements = article_soup.find_all('p')
+        article_text = headline.get_text(separator="\n")
+
+        for element in text_elements:
+            article_text += element.get_text(separator="\n") + "\n"
```

Comment:
concatenation is bad. use better way for building one string from many small ones

Reply:
Changed

Reply:
not enough. created new comment